// Initial proto file of this project

syntax = "proto3";

package demo_grpc;

import "myproto/big_data.proto";

message S_Response {
    string name     = 1;
    string street   = 2;
    string zip      = 3;
    string city     = 4;
    string country  = 5;

    float double_init_val = 6;

    Streaming server_data_collection = 7;
}

message C_Request {
    uint32 choose_area = 1;
    string name = 2;
    float init_val = 3;

    Streaming client_data_collection = 4;

    string area_field = 5;
}
